project.html

```diff
@@ -0,0 +1,512 @@
+                        <p class="reviews-country-name name"><a href="#">Rotterdam, Netherlands</a></p>
+                    </div>
+                    <div class="reviews-rating-viewing">
+                        <div id="rectangle">8,9</div>
+                        <p class="reviews-viewing"><a href="#">2016 reviews</a></p>
+                    </div>
+                </div>
+
+                <div class="reviews-review-2">
+                    <svg class="reviews-account">
+                        <use href="#reviews-account" />
+                    </svg>
+                    <div class="reviews-user">
+                        <p class="reviews-person">Adam Smith</p>
+                        <img class="reviews-person-country" src="images/reviews_images/usa_flag.svg" alt="USA" width="57" height="21">
+                    </div>
+                </div>
+
+                <div class="reviews-feedback">
+                    <p>I've been in this hostel for a month and here is the best hostel of all Excellent management of excellent and ethical staff and bedrooms Bathrooms are always clean and excellent Most importantly, those who work here are kind and good-natured.</p>
+                </div>
+
+            </div>
+        </div>
+        <div class="reviews-box">
+            <a href="#">
+            <div class="reviews-cost">From 100 USD</div>
+            <img class="reviews-image" src="images/reviews_images/Ourika.svg" alt="Ourika" width="400" height="400">
+            </a>
+                <div class="reviews-review">
+
+                <div class="reviews-review-1">
+                    <div class="reviews-name">
+                        <p class="reviews-hotel-name name _font-weight-bold"><a href="#">Rokoko Hotel</a></p>
+                        <p class="reviews-country-name name"><a href="#">Ourika, Marocco</a></p>
+                    </div>
+                    <div class="reviews-rating-viewing">
+                        <div id="rectangle">9,1</div>
+                        <p class="reviews-viewing"><a href="#">815 reviews</a></p>
+                    </div>
+                </div>
+
+                <div class="reviews-review-2">
+                    <svg class="reviews-account">
+                        <use href="#reviews-account" />
+                    </svg>
+                    <div class="reviews-user">
+                        <p class="reviews-person">Anika Messer</p>
+                        <img class="reviews-person-country" src="images/reviews_images/germany_flag.svg" alt="Germany" width="95" height="21">
+                    </div>
+                </div>
+
+                <div class="reviews-feedback">
+                    <p>Efficient, friendly, professional, and appropriately attentive. Great hotel in a great location. Easy access to all the surrounding historical sites. Staff is fantastic. Will definitely stay at this location during my next stay in Marocco.</p>
```

Comment:
If I add more text, element looks like this 
![image](https://user-images.githubusercontent.com/36200303/113487964-c041bf80-94c3-11eb-8e2d-ff1c7d0af6fd.png)